### DISCUSSION TITLE
Allow opening of comments when viewing image or video posts

### DISCUSSION THREAD

**Discussion by Pixxel123** [Docile-Alligator/Infinity-For-Reddit#975] [Ideas]
When image are clicked on there's the option to download, share or set as lock screen/wallpaper, but no option to open the comments section from that post view. It would be a welcome feature, maybe we could toggle the post name and which buttons to have visible, since wallpaper setting isn't that common, for example.

This view:

![Screenshot_20220517-173439_Infinity.jpg](https://user-images.githubusercontent.com/36304206/168863428-885fb82b-bfc5-4acd-b039-4295cd70a7fe.jpg)

**Comment by Wladefant** [comment]
@Pixxel123 I would agree that this is much more intuitive.
Everyone who is interested in this feature please consider upvoting the original post.
The wallpaper function is very uncommon and therefore I can not understand why it should be in the image menu

**Comment by Wladefant** [comment]
Is the icon right to the share icon an wallpaper icon?

**Comment by Pixxel123** [comment]
@Wladefant It's for setting wallpapers, yes.
